core/designsystem/src/iosMain/kotlin/io/github/droidkaigi/confsched2023/designsystem/theme/FontResource.kt

```diff
@@ -14,6 +14,9 @@ import org.jetbrains.compose.resources.resource
 val fontMap: HashMap<String, FontFamily?> = HashMap()
 
 // https://ishroid.medium.com/custom-font-loading-in-kmp-compose-multiplatform-2eb19865f61b
+// FIXME We initially planned to use this implementation on the iOS side.
+//However, when shared.swift is output by XCFramework, methods with Compose annotations are not output.
+//Therefore, this implementation cannot be referenced from the iOS side.
```

Comment:
This Actual method cannot be referenced from the iOS side.
In this case, SwiftGen shares the custom font by referencing the ttf file placed in commonMain.
Please let me know if this is wrong and I will try to fix it.